x/jwt.go

```diff
@@ -0,0 +1,15 @@
+package x
+
+import (
+	"encoding/base64"
+	"strings"
+)
+
+// Decode JWT specific base64url encoding with padding stripped
+func DecodeSegment(seg string) ([]byte, error) {
```

Comment:
Fosite does not offer a backwards compatible replacement of `DecodeSegment` function from `jwt-go` since it makes no use of it.  Therefore it was copied here directly from https://github.com/dgrijalva/jwt-go

Reply:
:+1: